trk/src/Screens/SetDetail/index.js

```diff
@@ -134,7 +104,7 @@ function SetDetail(props) {
               feedbackList.map((comment, index) => (
                 <View key={index} style={styles.commentContainer}>
                   <Text style={styles.commentText}>{comment.explanation}</Text>
-                  <Text style={styles.commentRating}>{renderRating(comment.rating)}</Text>
+                  <Text style={styles.commentRating}>{() => renderRating(comment.rating)}</Text>
```

Comment:
what was the issue here? Was there an issue with rendering?

Reply:
Yupp